skyfield/positionlib.py

```diff
@@ -152,18 +151,61 @@ def separation_from(self, another_icrf):
         c = dots(u1, u2)
         return Angle(radians=arccos(clip(c, -1.0, 1.0)))
 
-    def ecliptic_position(self):
-        """Compute J2000 ecliptic coordinates (x, y, z)"""
-        vector = _ECLIPJ2000.dot(self.position.au)
-        return Distance(vector)
+    def ecliptic_xyz(self, epoch=None):
```

Comment:
I think this name makes more sense than the old one! But to help Skyfield stay compatible, could you say `ecliptic_position = ecliptic_xyz` after defining the function so the old name keeps working? Thanks!

Reply:
This should pose no problem.